map/map.mbt

```diff
@@ -167,6 +167,42 @@ pub fn iter[K : Compare, V](self : Map[K, V], f : (K, V) -> Unit) -> Unit {
   }
 }
 
+/// Iterate over the key-value pairs with index.
+pub fn iteri[K : Compare, V](self : Map[K, V], f : (Int, K, V) -> Unit) -> Unit {
+  fn do_iteri(m : Map[K, V], f : (Int, K, V) -> Unit, i : Int) {
+    match m {
+      Empty => ()
```

Comment:
these bounds are unused. cc @Guest0x0, we may add a warning to warn unused bounds